technique/reflectometry/base.py

```diff
@@ -255,6 +255,59 @@ def slit_check(theta, footprint, resolution):
     print("s2vg={}".format(s2))
 
 
+def auto_height(laser_block: str, fine_height_block: str, target: float = 0.0, continue_if_NaN: bool=False):
```

Comment:
I think laser_block and target parameters could be named a bit better as it is not obvious immediately that they are offsets. Could be something like laser_offset_block and laser_offset target?